src/components/TopMenu.js

```diff
@@ -20,6 +22,16 @@ const TopMenu = () => {
     { href: "/settings", label: "Configuración" },
   ];
 
+  useEffect(() => {
+    let token = localStorage.getItem('token');
+    if (token) {
+     const datosToken = JSON.parse(atob(token.split('.')[1]));
```

Comment:
please us english instead of spanish also change to descriptive name : email or roles or something similar

Reply:
Understood.